corehq/apps/data_dictionary/tests/test_util.py

```diff
@@ -0,0 +1,60 @@
+from django.test import TestCase
+from mock import patch
+
+from corehq.apps.data_dictionary.models import CaseType, CaseProperty
+from corehq.apps.data_dictionary.util import generate_data_dictionary
+
+
+class GenerateDictionaryTest(TestCase):
+    domain = 'data-dictionary'
```

Comment:
I've started making test domains UUID's which makes it harder to get into a bad state, especially when using REUSE_DB.